examples/prompt_cli/.mirascope/versions/math_problem/0002_math_problem.py

```diff
@@ -0,0 +1,17 @@
+"""A basic prompt to solve a math problem."""
+
+from mirascope import tags
+from mirascope.openai import OpenAICall
+
+prev_revision_id = "0001"
+revision_id = "0002"
+
+
+@tags(["version:0002"])
```

Comment:
consider adding a version 3 that uses an extractor with two fields, one for reasoning, then one for just the answer?